File: 2_semester/Laba_6/task1_1.py
def task_1(vec, M):
	temp = [0 for i in range(M)]
	for item in vec:
		for i in range(M):
			item[i] = 10000000000 * (i+1);

def main():
	N = 3
	M = 3
	N, M = map(int,input().split())
	vec = []

	for i in range(N):
		vec.append([])
		for _ in range(M):
			vec[i].append(0)

	task_1(vec, M)

	for vect in vec:
		for item in vect:
			print(item, end = ' ')
		print()
	
main()

File: 2_semester/Laba_6/task1_1.cpp
#include <iostream>
#include <vector>
using namespace std;

void task_1(vector<vector<int64_t>> &vec, int M){
	vector<int64_t> temp (M);
	for (auto &item : vec){
		for (int i = 0; i < M; i++)
			temp[i] = 10000000000 * (i+1);
		item = temp;
	}
}

int main(){
	int N = 3, M = 3;
	cin >> N >> M;
	vector<int64_t> temp(M);
	vector<vector<int64_t>> vec (N);

	task_1(vec, M);

	for (auto vect : vec){
		for (auto item : vect)
			cout << item << ' ';
		cout << endl;
	}
}